Permission denied to github-actions[bot] for Code Formatting & Linting Workflow

Error on latest commit in Code Formatting & Linting Workflow
https://github.com/OSU-MC/Open-Response/actions/workflows/ci-formatting.yml
https://github.com/OSU-MC/Open-Response/actions/runs/18551635676/job/52880055600

Error 
```bash
Run git config user.name "github-actions[bot]"
[development 27de7eb] chore: auto-format with Prettier
 249 files changed, 54032 insertions(+), 51038 deletions(-)
remote: Permission to OSU-MC/Open-Response.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/OSU-MC/Open-Response/': The requested URL returned error: 403
Error: Process completed with exit code 128.
```

--- a/.github/workflows/ci-formatting.yml
+++ b/.github/workflows/ci-formatting.yml
@@ -11,6 +11,8 @@ jobs:
   formatting:
     name: Prettier, ESLint & Auto-format
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
 
     steps:
       - name: Check out code

